src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment }>
```

Comment:
You can pass in the function directly by doing `onClick={this.props.increment}` because the anonymous function is unnecessary here. Also if you are using the anonymous function then you would need to invoke increment at the end like this `button onClick={() => this.props.increment()}>`